.sequelizerc

```diff
@@ -1,4 +1,5 @@
-const path  = require('path')
+require ('@babel/register');
```

Comment:
Why do we need babel/register here?

Reply:
For sequelize to use ES6

![image](https://user-images.githubusercontent.com/52471122/78013913-76217d80-7347-11ea-8d82-7933af647937.png)

It's stated in the [documentation](https://sequelize.org/v5/manual/migrations.html#using-babel)